src/cards/ha-media_player-card.html

```diff
@@ -248,8 +248,20 @@
 
   playerObjChanged: function (playerObj) {
     var picture = playerObj.stateObj.attributes.entity_picture;
+    var dummy;
     if (picture) {
-      this.$.cover.style.backgroundImage = 'url(' + picture + ')';
+      dummy = document.createElement('IMG');
```

Comment:
I'm fine with not using iron-image. If we will need this in another place, we should extract this into a util function that returns a promise.

Oh and next time, you can use `dummy = new Image();`